.github/workflows/unit_test.yml

```diff
@@ -8,16 +8,16 @@ jobs:
     runs-on: ubuntu-latest
 
```

Comment:
can we use the reusable test workflow instead?

Reply:
this workflow has a step to download the build output artifact, which isn't in the reusable workflow. we need the `dist` directory for the tests to pass and it's nice if we can just download the artifact instead of running the `build` command again like the reusable workflow would do. so, I think it would be better to keep this workflow separate